src/Controls/tests/UITests/Tests/Issues/Issue19926.cs

```diff
@@ -16,7 +16,10 @@ public Issue19926(TestDevice device)
 	public void PropertiesShouldBeCorrectlyApplied()
 	{
 		this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.iOS, TestDevice.Mac, TestDevice.Windows });
+
 		_ = App.WaitForElement("boxView");
+		App.Click("button");
+		_ = App.WaitForElement("boxView2");
```

Comment:
Include a small delay here before take the snapshot to wait button ripple effect animation to complete.

Reply:
Done. I hope 500ms is enough